synchronization/src/main/java/de/cyface/synchronization/WiFiSurveyor.java

```diff
@@ -59,7 +75,7 @@
      * The <code>Account</code> currently used for data synchronization or <code>null</code> if no such
      * <code>Account</code> has been set.
      */
-    private Account currentSynchronizationAccount;
```

Comment:
Weil ich den Account in einem Test hart setzen muss:
_sync/androidTest/WifiSurveyor.java : testSetConnected()_

Siehe Kommentar:
// Instead of calling startSurveillance as in production we directly call it's implementation
        // Without the networkCallback or networkConnectivity BroadcastReceiver as this would make this test
        // flaky when the network changes during the test
        objectUnderTest.currentSynchronizationAccount = account;